From ce25f8076611ac4156339973eabaf3fd7a4d7ad4 Mon Sep 17 00:00:00 2001
From: Mattias Ellert <mattias.ellert@physics.uu.se>
Date: Sat, 19 Feb 2022 14:47:52 +0100
Subject: [PATCH] Enabling unit tests should not disable install of libraries
 and binaries

---
 CMakeLists.txt | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ec5c878..210a2c4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -80,14 +80,13 @@ ExternalProject_Add(gtest
 endif()
 enable_testing()
 add_subdirectory(test)
+endif()
 
-else()
 get_directory_property(TARGETS BUILDSYSTEM_TARGETS)
 install(
   TARGETS ${TARGETS} 
   LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
   RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} )
-endif()
 
 install(
   FILES src/scitokens.h
